docs/direct_mig_requirements.md

```diff
@@ -0,0 +1,26 @@
+# Direct Migration Requirements
+
+Direct Migration is available as a feature from Migration Toolkit for Containers (MTC) release 1.4.0. There are two parts of the Direct Migration - Direct 
+Volume Migration and Direct Image Migration. Direct Migration enables the migration of persistent volumes and internal
+images directly from the source cluster to the destination cluster without an intermediary replication repository (object storage)
+There are some requirements which are to be satisfied before you can use the Direct Migration feature of MTC.
+
+1. The clusters (source as well as destination) involved in the migration should expose their respective internal registries
+for external traffic.
+
+2. The remote source and destination clusters should be able to communicate via the OpenShift routes on port 443.
+
+3. The exposed registry route must be configured in the source and destination MigClusters, this can be done by specifying
+the `spec.exposedRegistryPath` field or via the MTC UI.
```

Comment:
Please confirm: Does the destination cluster require an exposed registry route? In the demos, I have only seen the registry route exposed for the source cluster.

Reply:
Update: Sergio confirmed that only the source cluster needs to have exposed registry route in MigCluster.

Reply:
The exposed registry route is needed for both - source and destination clusters, most of the times what happens is the destination cluster is also the host cluster (where the migration controller is deployed/resides) then in such a scenario the user does not need to specify the registry route, it is grabbed by default. But otherwise the user needs to specify the exposed registry route for source as well as destination clusters.

Reply:
Controller on remote cluster is one of our edge cases, not one of the use cases in downstream docs. Remote host cluster is covered by a Kbase article. I do not think exposed registry route needs to be specified for target cluster since we assume the controller is also installed on the target cluster. I'll think about it.

Reply:
@dymurray Can you confirm exactly when this field is required and on which/both clusters? My understanding is that this is not required for direct volume migrations, only direct image migrations. We should be specific about that.

Reply:
This is only required for Direct Image Migrations correct.

If the controller is running on the destination cluster (as downstream docs specify) then this value only needs to be exposed on the Source `MigCluster` resource that gets created.

Reply:
Updated the PR with an added clause (required only for Direct Image Migration and not Direct Volume Migration)